docs/documentation/make-first-prototype/link-pages-together.md

```diff
@@ -0,0 +1,34 @@
+# Link your pages together
+
+To take users from one page to another, either:
+
+- add HTML links to content
```

Comment:
I think we're only asking the user to update an existing link, not add a new one?

Reply:
This bullet list is about the user's options rather than what we're about to immediately tell them, so it feels accurate? Let me know what you think.